.changeset/config.json

```diff
@@ -7,10 +7,6 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": [
-    "ensnode",
-    "@ensnode/ensrainbow",
-    "@docs/ensnode",
-    "@docs/ensrainbow"
-  ]
+  "prettier": false,
```

Comment:
This is a new setting introduced with `changesets/config@3.1.0`:
https://github.com/changesets/changesets/releases/tag/%40changesets%2Fconfig%403.1.0